src/EmailAddressValidator.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+
+/**
+ * Validates Email address
+ */
+class EmailAddressValidator extends BaseValidator
+{
+    use Translation;
+
+    /**
+     * @var bool If MX check should be enabled
```

Comment:
Below you are using *done*, here *enabled*. Why the distinction?

Reply:
no particular reason, it has been changed to _enabled_ for both the cases.